fastlane/metadata/android/pt-PT/title.txt

```diff
@@ -1 +1 @@
-Catima
+Catima — Carteira de Cartão Fidelidade
```

Comment:
Leaving this MR open for a bit in the hope this gets updated to something fitting in 30 characters because Google Play limit (Weblate was misconfigured)

Reply:
Looks like this affects 3 existing languages:

```
$ for x in fastlane/metadata/android/*/title.txt; do t="$(< $x)"; (( ${#t} > 30 )) && echo $x; done
fastlane/metadata/android/ast/title.txt
fastlane/metadata/android/hr/title.txt
fastlane/metadata/android/pl-PL/title.txt
```

Reply:
Yup, going to have to wipe the suffixes if it doesn't get updated because otherwise the metadata upload to Google Play will fail